### DISCUSSION TITLE
Comment source location tooltip

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#4624] [Ideas]
**Description:**

It would be nice at sompoint to provide additional context on where the user will jump when they hover on the comment snippet.

![source tooltip](https://user-images.githubusercontent.com/254562/138542883-a03c3f09-244c-444b-89c1-724d8197e64e.png)

https://www.figma.com/file/SSr1ljyzF0lCXTL7BX1Whh/Replay-Design-Doc?node-id=7746%3A48940

**Comment by jonbell-lot23** [comment]
March 15, 2022:

I love this idea. It's clear we have a big group of feature requests around hovering, modifier key clicking, etc to get a bit more information before diving in further. Might make for a good project in its own right.
